examples/solidification_region_stl/input.yaml

```diff
@@ -3,15 +3,15 @@ steps:
     class: solidification_region_stl
     application: additivefoam
     configure:
-      coarse: 160.0e-6
+      coarse: 0.64e-3
       rx: 1.0e-3
       ry: 1.0e-3
-      rz: 300.0e-6
-      pad-xy: 0.75e-3
-      pad-z: 0.4e-3
-      pad-sub: 2.5e-3
-      refine-layer: 3
-      refine-region: 1
+      rz: 0.5e-3
+      pad-xy: 2.0e-3
+      pad-z: 1.0e-3
+      pad-sub: 1.0e-3
+      refine-layer: 2
+      refine-region: 3
```

Comment:
Changed to be more tractable with the current example files. The example STL and layer numbers for the example scan paths changed, so the location of the region and appropriate meshing values changed.

I'm including this in this PR because I found it necessary to test the refactored `solidification_region_stl` app.